views/home.js

```diff
@@ -51,7 +51,7 @@ export default function HomePageScene({navigator, route}: ScenePropsType) {
     >
       <StatusBar
         barStyle='light-content'
-        backgroundColor={c.gold}
+        backgroundColor={c.olevilleGold}
```

Comment:
Gah. That's what I get for testing on two devices and not refreshing both. Fixing.